python-sdk/src/astro/files/base.py

```diff
@@ -145,7 +145,7 @@ def resolve_file_path_pattern(
     location = create_file_location(path_pattern, conn_id)
 
     files = [
-        File(
```

Comment:
I could fix this issue by adding `attrs` to additional deps. See [here](https://github.com/astronomer/astro-sdk/pull/780/commits/7c75a20f992927842a64dd1a2a8eab402cdee333).

Reply:
@utkarsharma2